publisher-sdk/src/main/java/com/criteo/publisher/adview/MraidInteractor.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher.adview
+
+import android.webkit.WebView
+import com.criteo.publisher.annotation.OpenForTesting
+
+@OpenForTesting
+internal class MraidInteractor(private val webView: WebView) {
+
+  fun notifyReady() {
+    callOnMraidObject("notifyReady()")
+  }
+
+  fun notifyError(message: String, action: String? = null) {
+    val actionString = if (action == null) "" else ", \"$action\""
+    callOnMraidObject("notifyError(\"$message\"$actionString)")
```

Comment:
For now you have a single JS call, but maybe when you have plenty of those, you could think about a utility method to transform a list of args into JS args. Like:
- given a string `foo` -> `"foo"`
- given an int `42` -> `42`
- given a null -> `undefined`

And joining the args with comma. So you won't have to deal (and maybe forget) the quotes around strings or handling the nulls

Reply:
Great idea, updated my code